package.json

```diff
@@ -5,9 +5,11 @@
     "url": "git://github.com/asyncapi/studio.git"
   },
   "scripts": {
+    "lint": "eslint",
```

Comment:
Why do you need eslint here?

Reply:
so that testing workflow can run the linter and ensure code consistency.